### PR TITLE
Match dependency levels with operators

### PR DESCRIPTION
**Pull request by mfojtik**
See the glide commit for details. This brings us on pair with operator repos.

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -6,7 +6,8 @@ import:
 - package: k8s.io/apiserver
   version: kubernetes-1.13.4
 - package: k8s.io/apiextensions-apiserver
-  version: kubernetes-1.13.4
+  repo: https://github.com/openshift/kubernetes-apiextensions-apiserver
+  version: 6e2b3eb1b401beaf836f640f73f730f6da7aff2f # pinned to get nullable patch
```
Comment (deads2k) [inline glide.yaml]: I would track a branch instead.  Also, why do this when the ugly should be hidden in the new repo?

File: glide.yaml [hunk 2]
```diff
@@ -18,10 +19,10 @@ import:
 
 # crd-schema-gen
 - package: sigs.k8s.io/controller-tools
-  version: 43466124052c1a1aa7d8fd33624b00bc111fb7cf
-  repo: https://github.com/openshift/kubernetes-sigs-controller-tools.git
+  repo: https://github.com/openshift/kubernetes-sigs-controller-tools
+  version: master
 - package: k8s.io/gengo
-  version: 4242d8e6c5dba56827bb7bcf14ad11cda38f3991
+  version: master
```
Comment (deads2k) [inline glide.yaml]: this is wrong.  Choose a specific level.

File: glide.yaml [hunk 3]
```diff
@@ -18,10 +19,10 @@ import:
 
 # crd-schema-gen
 - package: sigs.k8s.io/controller-tools
-  version: 43466124052c1a1aa7d8fd33624b00bc111fb7cf
-  repo: https://github.com/openshift/kubernetes-sigs-controller-tools.git
+  repo: https://github.com/openshift/kubernetes-sigs-controller-tools
+  version: master
```
Comment (deads2k) [inline glide.yaml]: this looks unlikely to keep working well.  If this is a fork, you need a branch.  If we really want to track a master, pin to sha.

Comment (sttts) [reply glide.yaml]: https://github.com/sttts/controller-tools/tree/master-1.13-rebase – how should we call it?

File: glide.yaml (hunk 1 above)
Comment (mfojtik) [reply glide.yaml]: Tracking branch now with TODO about moving this to new repo once @damemi is ready.

File: glide.yaml (hunk 2 above)
Comment (mfojtik) [reply glide.yaml]: added specific commit.

File: glide.yaml (hunk 3 above)
Comment (mfojtik) [reply glide.yaml]: Tracking branch now with standard name.

**Comment by mfojtik** [comment]
(re-adding label manually as I just removed extra new line in glide.yaml ;-)

**Pull request merged**